src/java/picard/util/MetricsDoclet.java

```diff
@@ -79,6 +79,9 @@ public static boolean start(final RootDoc root) {
                         firstSentence(doc) + "</li>");
         }
         out.println("</ol>");
+        out.println("Note: Metrics labeled as percentages (with 'percent' in the full metric name or 'PCT' in the name given in " +
```

Comment:
No I mean so that the formatting will be ok in the generated html, which is independent of how the code looks in IntelliJ. Text that doesn't have any html tags should come out as paragraph-styled by default, but I'd rather not take any chances.